cmd/root_test.go

```diff
@@ -1348,6 +1348,41 @@ func TestQuitQuitQuit(t *testing.T) {
 	}
 }
 
+func TestQuitQuitQuitHTTPGet(t *testing.T) {
+	c := NewCommand(WithDialer(&spyDialer{}))
+	c.SilenceUsage = true
+	c.SilenceErrors = true
+	c.SetArgs([]string{"--quitquitquit", "--admin-port", "9193", "my-project:my-region:my-instance"})
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	errCh := make(chan error)
+	go func() {
+		err := c.ExecuteContext(ctx)
+		errCh <- err
+	}()
+
+	resp, err := tryDial("GET", "http://localhost:9192/quitquitquit")
```

Comment:
Maybe we pull the port out as a constant to avoid this kind of mistake (9192 here, 9193 above)